Hardware/Hardware/Printer/ReceiptContent.swift

```diff
@@ -57,3 +58,15 @@ public extension ReceiptContent {
         )
     }
 }
+
+public extension ReceiptContent {
+    static let pointsPerInch: Int = 72
+
+    /// Returns the preferred page size for a receipt in points. There are 72 points per inch.
+    /// In the future, we could calculate this based on the receipt content. For now, let's
+    /// just return a size that should accomodate the vast majority of receipts.
+    ///
+    var preferredPageSizeForPrinting: CGSize {
+        return CGSize(width: 4 * ReceiptContent.pointsPerInch, height: 10 * ReceiptContent.pointsPerInch)
```

Comment:
I understand this leaves enough vertical space for receipts to grow, but can we at least log a ticket to estimate a height, please?

Reply:
```suggestion
        return CGSize(width: 4 * ReceiptContent.pointsPerInch, height: 6 * ReceiptContent.pointsPerInch)
```

I tried this as an experiment. It didn't seem to get used/impact the printing on an A4 printer at all, while on the receipt printer it rendered out as split pages in the modal. That made no impact on what got printed on the roll printer though, it just printed the content and didn't reel out a load of extra paper afterwards. I also tried with a height of 1, and even though the modal looked silly (receipt split into 1" chunks) it still printed correctly without wasting paper.

https://user-images.githubusercontent.com/2472348/144453950-2928ecd9-8b20-4045-bbfe-767243c343a9.mp4

![IMG_5957](https://user-images.githubusercontent.com/2472348/144454005-3630fd3a-25ec-4d69-8986-942d4dd0f1bd.jpg)